src/Pages/PeoplePage.tsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PeopleTable } from '../components/PeopleTable/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(data => {
+        setPeople(data);
+      })
+      .catch(() => setIsError(true))
```

Comment:
Consider providing more detailed error information to the user. Currently, the error message is generic ('Something went wrong'). If possible, include specific error details to help users understand what went wrong.